.github/workflows/publish-release.yml

```diff
@@ -88,3 +113,21 @@ jobs:
           npm-token: ${{ secrets.NPM_TOKEN }}
         env:
           SKIP_PREPACK: true
+
+  publish-release-to-gh-pages:
+    needs: publish-npm
+    name: Publish docs to `${{ github.ref_name }}` directory of `gh-pages` branch
```

Comment:
`github.ref_name` is usually set according to the event that triggered the workflow. In this case, that event won't have the correct refname set because it hasn't been tagged yet at that point. We'll; have to get the version name from somewhere, like `package.json` maybe.

Reply:
Ricky has a script for this already: https://github.com/MetaMask/action-publish-release/blob/a0e96f152a64664437d4dc5bfc381f42fa75978a/action.yml#L12-L19